platform/broadcom/sai.mk

```diff
@@ -0,0 +1,9 @@
+BRCM_SAI = libsaibcm_2.0.3.7~20161214010054.112_amd64.deb
+$(BRCM_SAI)_PATH = $(PLATFORM_PATH)/sdk
+BRCM_SAI_DEV = libsaibcm-dev_2.0.3.7~20161214010054.112_amd64.deb
+$(BRCM_SAI_DEV)_PATH = $(PLATFORM_PATH)/sdk
```

Comment:
Use add_derived_package for dev packages, like here:
https://github.com/Azure/sonic-buildimage/blob/master/rules/sairedis.mk#L10
It will copy all info, like path etc. and it will be copied with main package automatically, so you don't need to add it to SONIC_COPY_DEBS